src/routes/feedback/FeedbackVariant.js

```diff
@@ -1,20 +1,23 @@
 // @flow
 
-import type { FeedbackType } from '@integreat-app/integreat-api-client/endpoints/createFeedbackEndpoint'
+import type { FeedbackType, FeedbackCatetory } from '@integreat-app/integreat-api-client/endpoints/createFeedbackEndpoint'
 
 class FeedbackVariant {
   label: string
   language: string
   city: string
   feedbackType: FeedbackType
+  feedbackCategory: ?FeedbackCatetory
```

Comment:
This type probably should be optional. I think you have to select something in the UI.